deploy/crds/logging.openshift.io_clusterloggings_crd.yaml

```diff
@@ -0,0 +1,795 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
we dont need this from the sdk as we don't deploy using this mechanism but via OLM.  the content needs to be merged into the OLM manifest

Reply:
+1 my use of operator-sdk was naive, will figure out how to properly integrate this with OLM manifests before I merge. https://issues.redhat.com/browse/LOG-664